package.json

```diff
@@ -25,8 +25,8 @@
     "url": "https://github.com/slackapi/slack-github-action/issues"
   },
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=6.4.1"
+    "node": ">=16.0.0",
+    "npm": ">=8.15.0"
```

Comment:
Nit and may not matter: It looks like `node@16` ships with `npm@7.10.0`
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/21328/190276367-90739c50-5a95-4e5e-b0a9-8d09252ab508.png">
Reference: https://nodejs.org/en/download/releases/

Reply:
Ah good shout. Fixed!